repos/spack_repo/builtin/packages/darshan_util/package.py

```diff
@@ -14,14 +14,14 @@ class DarshanUtil(AutotoolsPackage):
     log files produced by Darshan (runtime)."""
 
     homepage = "https://www.mcs.anl.gov/research/projects/darshan/"
-    url = "https://web.cels.anl.gov/projects/darshan/releases/darshan-3.4.0.tar.gz"
```

Comment:
Same here. This property is used to find new versions with `spack versions` and `spack checksum` commands.